.ci/static-checks.sh

```diff
@@ -891,6 +892,43 @@ static_check_xml()
 	done
 }
 
+static_check_shell()
+{
+	local all_scripts
+	local scripts
+
+	all_scripts=$(find . \( -name "*.sh" -o -name "*.bash" \) | grep -v "/vendor/" | sort || true)
+
+	if [ "$specific_branch" = "true" ]
+	then
+		info "Checking all scripts in $branch branch"
+		scripts="$all_scripts"
+	else
+		info "Checking local branch for changed scripts only"
+
+		local scripts_status
+		scripts_status=$(get_pr_changed_file_details || true)
+		scripts_status=$(echo "$scripts_status" | grep -E "\.(sh|bash)$" || true)
+
+		scripts=$(echo "$scripts_status" | awk '{print $NF}')
+	fi
+
+	[ -z "$scripts" ] && info "No scripts to check" && return 0
+
+	local script
+
+	for script in $scripts
+	do
+		info "Checking script file '$script'"
+
+		local ret
+
+		{ chronic bash -n "$script"; ret=$?; } || true
```

Comment:
y'know, sometimes I have `man bash`, coz finding specifics in there can be so hard... but, yes, `-n`, interesting :-) the cheap and dirty shellcheck...

Reply:
As mentioned on the issue, it's basic but doesn't require us to modify our scripts like `shellcheck` will and will find basic issues. Maybe when we have a bunch of free time (cough...), we can update the entire corpus of `~230` scripts for `shellcheck` :)